contract/tests/test_contract.py

```diff
@@ -248,15 +252,15 @@ def test_contract_invoice_followers(self):
         self.acct_line.recurring_rule_type = 'daily'
         self.contract.pricelist_id = False
         self.contract.message_subscribe(
-            partner_ids=self.contract.partner_id.ids,
-            subtype_ids=self.env.ref(
-                'contract.mail_message_subtype_invoice_created'
-            ).ids
+            partner_ids=[self.contract.partner_id.id],
+            subtype_ids=[
+                self.env.ref('contract.mail_message_subtype_invoice_created').id
+            ]
         )
         self.contract.recurring_create_invoice()
         invoice_daily = self.contract._get_related_invoices()
         self.assertTrue(invoice_daily)
-        self.assertEqual(len(invoice_daily.message_partner_ids.ids), 2)
+        self.assertEqual(len(invoice_daily.message_partner_ids.ids), 1)
```

Comment:
Why this change?

Reply:
IMO because before, more followers were added than they should.